website/docs/components/table/partials/code/how-to-use.md

```diff
@@ -164,7 +162,83 @@ You can optionally also indicate that the column defined in `@sortBy` should be
 </Hds::Table>
 ```
 
-#### More Examples: internationalized column headers, overflow menu dropdown
+#### Custom sort callback
+
+To implement a custom sort callback on a columns, (1) add a custom function as the value for `sortingFunction` in the column hash, and (2) include a custom `onSort` action in your table invocation. This is useful for cases where the key might not be A-Z or 0-9 sortable by default, i.e., status, and you’re otherwise unable to influence the shape of the data in the model. Here is an example, code truncated for clarity:
```

Comment:
```suggestion
To implement a custom sort callback on a columns, (1) add a custom function as the value for `sortingFunction` in the column hash, and (2) include a custom `onSort` action in your table invocation to track the sorting order and use it in the custom sorting function. This is useful for cases where the key might not be A-Z or 0-9 sortable by default, i.e., status, and you’re otherwise unable to influence the shape of the data in the model. Here is an example, code truncated for clarity:
```

Reply:
Also, should we use a `<ol>` element here, instead of `(1)/(2)`?